### PR TITLE
gnutls: removed Nikos as primary contact and add Daiki

### PR DESCRIPTION
**Pull request by nmav**
Signed-off-by: Nikos Mavrogiannopoulos <n.mavrogiannopoulos@gmail.com>

### CODE REVIEW

**Comment by nmav** [comment]
@ueno is it ok to add you as primary contact?

**Comment by ueno** [comment]
Sure.

File: projects/gnutls/project.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 homepage: "https://www.gnutls.org"
 language: c++
-primary_contact: "n.mavrogiannopoulos@gmail.com"
+primary_contact: "daiki.ueno@gmail.com"
 auto_ccs:
   - "daiki.ueno@gmail.com"
```
Comment (inferno-chromium) [inline projects/gnutls/project.yaml:5]: Probably want to change this auto_cc to n.mavrogiannopoulos@gmail.com ?

**Review by inferno-chromium — CHANGES_REQUESTED** [review]

File: projects/gnutls/project.yaml (hunk 1 above)
Comment (nmav) [reply projects/gnutls/project.yaml:5]: I do not think that's necessary.

Comment (inferno-chromium) [reply projects/gnutls/project.yaml:5]: Ok can you remove the duplicate email from auto_ccs field.

**Comment by nmav** [comment]
Thanks, done!

**Review by inferno-chromium — APPROVED** [review]

**Pull request merged**